modules/util.py

```diff
@@ -14,8 +15,16 @@
 import modules.sdxl_styles
 
 LANCZOS = (Image.Resampling.LANCZOS if hasattr(Image, 'Resampling') else Image.LANCZOS)
+
+
+# Regexp compiled once. Matches entries with the following pattern:
+# <lora:some_lora:1>
+# <lora:aNotherLora:-1.6>
+LORAS_PROMPT_PATTERN = re.compile(".*<lora:(.+):([-+]?(?:\d*\.*\d*))>.*")
```

Comment:
This incorrectly matches e.g. `<lora:foo:1..2>`, `<lora:bar:.>`, `<lora:baz:+>` and `<lora:quux:>`. Also, some backtracking can be avoided, e.g.:

```python
LORA_PROMPT_PATTERN = re.compile(r".* <lora : ([^:]+) : ([+-]? (?: (?:\d+ (?:\.\d*)?) | (?:\.\d+)))> .*", re.X)
```

This matches, e.g.:

- `<lora:foo:1>`
- `<lora:bar:-1.2>`
- `<lora:baz:1.>`
- `<lora:quux:+.2>`

Reply:
I will make the change and include those cases in the unit test. Thanks @chocolateboy